src/software/thunderscope/common/proto_configuration_widget.py

```diff
@@ -155,3 +162,10 @@ def build_proto(self, message, current_attr=None):
                     exec(f"{current_attr}.{key} = {value}")
             else:
                 self.build_proto(value, f"{current_attr}.{key}")
+
+    def __first_shot(self):
+        """Send the current configuration to the AI after a delay"""
+        time.sleep(ProtoConfigurationWidget.DELAYED_CONFIGURATION_TIMEOUT_S)
+        self.on_change_callback(
+            str(self.proto_to_configure), None, self.proto_to_configure
+        )
```

Comment:
Curious why we need to call `on_change_callback` at the start?

Reply:
Also worth noting that full system echos back the config we sent it as a form of confirmation that it's received it. So maybe we could use that somehow instead of having an arbitrary delay? 

https://github.com/UBC-Thunderbots/Software/blob/8879ef476da31e7f716e9e662ad88c07e0637089/src/software/backend/unix_simulator_backend.cpp#L52-L59

Reply:
We need something to send the config at the beginning of Thunderscope so that `RobotCommuncation` knows what to bind to.

AI launches with the default params defined in the proto file